### PR TITLE
 Add configuration file for probot-stale - https://github.com/probot/stale

### PR DESCRIPTION
**Pull request by amedyne**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# pulls:
+#   daysUntilStale: 30
+#   markComment: >
+#     This pull request has been automatically marked as stale because it has not had
+#     recent activity. It will be closed if no further activity occurs. Thank you
+#     for your contributions.
+
+# issues:
+#   exemptLabels:
+#     - confirmed
```
Comment (jonalmeida) [inline .github/stale.yml:61]: We probably want to mark some issues as "keep" that we know are still valid.

```suggestion
issues:
  exemptLabels:
    - keep
```

Comment (amedyne) [reply .github/stale.yml:61]: We'll use "pin" for this. As we discussed in the team for this. That's why I had "pin" and "feature request" as exempt labels.

Comment (jonalmeida) [reply .github/stale.yml:61]: Any label is fine, but we need to un-comment the command for them to take effect. 🙂

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - "feature request 🌟"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
```
[the remark this answers is not in the document]
Comment (amedyne) [reply .github/stale.yml:25]: I prefer we close them for now. We can go back and open them as we go thru fixing or changing related areas or bug.

File: .github/stale.yml (hunk 1 above)
Comment (amedyne) [reply .github/stale.yml:61]: I thought that was what lines 14-16 are for? 😃

Comment (amedyne) [reply .github/stale.yml]: see updated Issue exempt labels/tags.

**Review by jonalmeida — APPROVED** [review]
I ran this file through a [YAML validator](https://yamlvalidator.com/) and found some small nits to fix, everything else looks good to me! 🙂

**Pull request closed without merging**